candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -3,15 +3,12 @@ RotateKubeletServerCertificate default is true, but CIS becnhmark wants it to be
 https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 */}}
 {{- $featureGates := list "TopologyAwareHints=true" "RotateKubeletServerCertificate=true" | join "," }}
-{{- if semverCompare ">= 1.26" .clusterConfiguration.kubernetesVersion }}
+{{- if semverCompare "< 1.30" .clusterConfiguration.kubernetesVersion }}
```

Comment:
let union ValidatingAdmissionPolicy and AdmissionWebhookMatchConditions 
under one `if`